tests/foreman/cli/test_contentview.py

```diff
@@ -1133,9 +1174,47 @@ def test_positive_promote_rh_and_custom_content(self):
 
         @assert: Content view can be promoted
 
-        @caseautomation: notautomated
-
+        @CaseLevel: Integration
         """
+        # Enable RH repo
+        self.create_rhel_content()
+        # Create custom repo
+        new_repo = make_repository({
+            'product-id': make_product({
+                'organization-id': self.rhel_content_org['id']})['id']
+        })
+        # Sync custom repo
+        Repository.synchronize({'id': new_repo['id']})
+        # Create CV
+        new_cv = make_content_view({
+            'organization-id': self.rhel_content_org['id'],
+        })
+        # Associate repos with CV
+        ContentView.add_repository({
+            'id': new_cv['id'],
+            'repository-id': self.rhel_repo['id'],
+        })
+        ContentView.add_repository({
+            'id': new_cv['id'],
+            'repository-id': new_repo['id'],
+        })
+        # Publish a new version of CV
+        ContentView.publish({'id': new_cv['id']})
+        new_cv = ContentView.info({'id': new_cv['id']})
+        env1 = make_lifecycle_environment({
+            'organization-id': ContentViewTestCase.rhel_content_org['id'],
+        })
+        # Promote the Published version of CV to the next env
+        ContentView.version_promote({
+            'id': new_cv['versions'][0]['id'],
+            'to-lifecycle-environment-id': env1['id'],
+        })
+        new_cv = ContentView.info({'id': new_cv['id']})
+        environment = {
+            'id': env1['id'],
+            'name': env1['name'],
+        }
+        self.assertIn(environment, new_cv['lifecycle-environments'])
```

Comment:
@sghai Missed that part, thanks for raising it. I have a question though - should i just add the filter and add no new assertions (like docstring asks) or i should assert that filter actually worked and CVV has filtered packages? If second option is preferred, then how should i fetch the packages list? `content-view info` or `content-view version info` do not return included packages, maybe there's some other command?